package.json

```
@@ -41,31 +41,40 @@
     "dev:generate:vercel": "NITRO_PRESET=vercel nuxi generate playground",
     "dev:build": "nuxi build playground",
     "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
-    "dev:preview:build": "npm run dev:build && node playground/.output/server/index.mjs",
-    "dev:preview:generate": "npm run dev:generate && npx serve playground/dist",
+    "dev:preview:build": "nr dev:build && node playground/.output/server/index.mjs",
+    "dev:preview:generate": "nr dev:generate && serve playground/dist",
     "prepublishOnly": "npm run prepack",
     "release": "bumpp && npm publish",
     "lint": "eslint .",
-    "lint-fix": "nr lint --fix"
+    "lint-fix": "nr lint --fix",
+    "test:build:serve": "PORT=4173 node playground/.output/server/index.mjs",
+    "test:generate:serve": "PORT=4173 serve playground/dist",
+    "test:build": "nr dev:build && TEST_BUILD=true vitest run && TEST_BUILD=true playwright test",
+    "test:generate": "nr dev:generate && vitest run && playwright test",
+    "test": "nr test:build && nr test:generate",
+    "test:with-build": "nr dev:prepare && nr prepack && nr test"
   },
   "peerDependencies": {
     "@nuxt/kit": "^3.0.0 || ^3.1.0",
```

should this be bumped?

---

Can you update the pr?

---

I guess my question is whether you specifying 3.0.0 and 3.1.0 separately to work around a package manager issue? It feels like we shouldn't need to specify each minor version with a caret dependency range...

So likely we should either replace with:

```suggestion
    "@nuxt/kit": "^3.0.0",
```

or the much more verbose (if there's a reason for it):

```suggestion
    "@nuxt/kit": "^3.0.0 || ^3.1.0 || ^3.2.0 || ^3.3.0 || ^3.4.0",
```